vignettes/rhino-fluent.Rmd

```diff
@@ -0,0 +1,556 @@
+---
+title: "Tutorial: Build a Rhino application using shiny.fluent"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEncoding{UTF-8}
+  %\VignetteIndexEntry{Tutorial: Build a Rhino application using shiny.fluent}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+## 1. Hello World with `rhino` and `shiny.fluent`
+
+### Getting started with our development environment
+
+Let us first install and create our `rhino` project structure.
+
+To install `rhino`, please run the following on a R console in your working directory
+
+```         
+install.packages("rhino")
+```
+
+Now that `rhino` is installed, let us use it to initiate a project which will give us a structure
+
+```         
+rhino::init()
+```
+
+You will notice that the working directory now has a proper folder structure along with some other files.
+Among those, you will also notice that `renv` is being used to manage the packages/dependencies for the project.
+So, let us now install `shiny.fluent` into the project.
+
+```         
+renv::install("shiny.fleunt")
+```
+
+Using renv for installation benefits us.
+In particular, this means that the local cache of package installations is used when possible.
+This helps to avoid re-downloading packages that have already been downloaded before, and re-compiling packages from source when a binary copy of that package is already available.
+
+Now, head over to the `dependencies.R` file and add `library(shiny.fluent)` to it.
+Upon saving the modified file, please take a snapshot of the dependencies so that the `renv.lock` file is updated.
+
+```         
+renv::snaphot()
+```
+
+Maintain this process whenever we need to install a new package in the project henceforth.
+
+### Our first development on the app
+
+-   Open the `app/main.R` file.
+
+-   You will notice that currently it does not have any usage of `shiny.fluent` package to generate the text - "Hello".
+    So let us introduce `shiny.fluent` and use it in our code to do the same action!
+
+```         
```

Comment:
Please add information about the edited file to the snippet (follow the style in e.g. Rhino main tutorial).

Reply:
fixed in latest commit